designs/inter-ident-verify.md

```diff
@@ -0,0 +1,78 @@
+# Interpersonal Identity Verification
+### Inspired by GitHub user [Charcol](https://github.com/kanidm/kanidm/issues/337)
+
+___
+## Motivation
+Social engineering scams over the phone can be used to manipulate users into unknowingly giving up information to third parties with malicious intent. The primary issue is that one party does not know the identity of the other, while the other already knows or doesn't particularly care. If we can allow at least one party to verify the identity of the other, then we can solve this issue.
+___
+## Existing solutions
+The issue of verifying identities when a company initiates contact via phone is still mostly unsolved, so the existing solution is to avoid phone calls from companies, and only trust them if the user is the caller and the company is the receiver. While this solution works, it doesn't address the issue that scammers will still scam through phone calls, and that if a user picks up the phone, they have no reasonable method to verify the authenticity of the call. For example, it is unsafe to trust anything that an unknown caller claims, such as going to a website to "verify" identities.
+___
+## Proposed solution
+Many companies have various branches that operate different parts of the company. For example, take the following parties:
+* U, the individual user
+* A, the big name corporation the user knows of
+* S, a branch of A that handles tech support, and is trusted by A
+
+If A wanted to contact U, the CEO wouldn't call them directly. Instead, they would reach out through S, which lives in a tightly vetted whitelist of callers approved by A. S calls U, and U must verify that S is a trusted branch of A, with only the knowledge that S supposedly is trusted by A. U doesn't care about the exact identity of the caller, only that they are trusted by A, since no scam company could make A's tightly vetted whitelist of trusted callers. Since they're not contacting A directly, there are four key connections that must be verified:
+1. U needs to know the S is trusted by A.
```

Comment:
I think these letter abreviations make it less clear. User, Support, Corp works?